docs/components/attention/vue.md

```diff
@@ -62,13 +86,69 @@ const target = ref(null)
 const showing = ref(false)
 </script>
 <template>
-  <w-box neutral class="h4" ref="target" @mouseenter="showing = true" @mouseleave="showing = false">Hover over me</w-box>
-  <w-attention tooltip bottom :target-el="target?.$el" v-model="showing">
-    <p>Hello Warp!</p>
+  <w-box
+    neutral
+    as="h4"
+    ref="target"
+    @mouseenter="showing = true"
+    @mouseleave="showing = false"
+    @keydown.escape="showing = false"
+    @focus="showing = true"
+    @blur="showing = false"
+    tabindex="0"
+  >
+    <button
+      aria-describedby="tooltip-bubbletext"
+      aria-expanded="true"
+      type="button"
+    >
+      Hover over me
+    </button>
+  </w-box>
```

Comment:
I think we can replace w-box with w-button. Here it doesn't matter how it looks, it's just a code example, but the simpler the example, the better.